fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -127,6 +142,32 @@ public void contextInitialized(ServletContextEvent event) {
                 log.info("Bypassing Kafka notification init.");
             }
 
+            // If NATS notifications are enabled, start up our NATS notification publisher.
+            Boolean natsEnabled = fhirConfig.getBooleanProperty(PROPERTY_NATS_ENABLED, Boolean.FALSE);
+            if (natsEnabled) {
+                // Retrieve the cluster ID.
+                String clusterId = fhirConfig.getStringProperty(PROPERTY_NATS_CLUSTER, DEFAULT_NATS_CLUSTER);
+                // Retrieve the channel name.
+                String channelName = fhirConfig.getStringProperty(PROPERTY_NATS_CHANNEL, DEFAULT_NATS_CHANNEL);
+                // Retrieve the NATS client ID.
+                String clientId = fhirConfig.getStringProperty(PROPERTY_NATS_CLIENT, DEFAULT_NATS_CLIENT);
+                // Retrieve the server URL.
+                String servers = fhirConfig.getStringProperty(PROPERTY_NATS_SERVERS);
+
+                // Gather up the NATS TLS properties.
+                Properties tlsProps = new Properties();
+                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED).toString());
+                tlsProps.setProperty("truststore", fhirConfig.getStringProperty(PROPERTY_NATS_TRUSTSTORE));
+                tlsProps.setProperty("truststore-pw", fhirConfig.getStringProperty(PROPERTY_NATS_TRUSTSTORE_PW));
+                tlsProps.setProperty("keystore", fhirConfig.getStringProperty(PROPERTY_NATS_KEYSTORE));
+                tlsProps.setProperty("keystore-pw", fhirConfig.getStringProperty(PROPERTY_NATS_KEYSTORE_PW));
```

Comment:
this is system specific and not tenant specific.  Best to change the docs to reflect that. 
I also suggest using a seamless naming convention, truststore-pw and truststore_pwd are different names.

Reply:
docs are updated.  naming can be addressed at some future date if needed.